lighthouse-core/audits/network-server-latency.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * @license Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const Audit = require('./audit');
+const i18n = require('../lib/i18n/i18n.js');
+const NetworkAnalysisComputed = require('../computed/network-analysis.js');
+
+const UIStrings = {
+  /** Descriptive title of a Lighthouse audit that tells the user the server latencies observed from each origin the page connected to. This is displayed in a list of audit titles that Lighthouse generates. */
+  title: 'Server Latencies',
+  /** Description of a Lighthouse audit that tells the user that server latency can effect their website's performance negatively. This is displayed after a user expands the section to see more. No character length limits. 'Learn More' becomes link text to additional documentation. */
+  description: 'Server latencies can impact web performance. ' +
+    'If the server latency of an origin is high, it\'s an indication the server is overloaded ' +
```

Comment:
I think this description should also indicate how the RTT is factored in. Are these values JUST the backend latency or the BL + RTT?

Reply:
I'm not sure how to put this without unnecessarily confusing folks. I'm not confident that the number of people who will understand this subtlety is very high (or the people that will know what to do with this audit, honestly)